src/App.js

```diff
@@ -1,34 +0,0 @@
-import { Fragment, useState } from "react";
-
-import Title from "./components/Layout/Title";
-import Words from "./components/Words/Words";
-import Bravo from "./components/Bravo/Bravo";
-
-function App() {
-  const [modalIsShown, setModalIsShown] = useState(false);
-  const [clearedCards, setClearedCards] = useState({});
-
-  const closeModalHandler = () => {
-    setModalIsShown(false);
-    setClearedCards({});
-    window.location.reload();
```

Comment:
Reloading is fine here, but we can also have some reset function that re-initializes everything.

Just a design choice.